code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -572,5 +572,3 @@
 /mob/living/carbon/human/proc/delete_equipment()
 	for(var/slot in get_all_slots())//order matters, dependant slots go first
 		qdel(slot)
-	for(var/obj/item/I in l_hand, r_hand)

```

Comment:
are you SURE the above also returns the hands.

Reply:
get_all_slots returns l_hand and r_hand